assets/src/styles/blocks.scss

```diff
@@ -9,6 +9,7 @@
 @import "blocks/SplitTwoColumns";
 @import "blocks/Submenu";
 @import "blocks/Cookies";
+@import "blocks/ShareButtons/ShareButtonsStyles";
```

Comment:
This needs to be set up with code splitting if we want to merge it as a beta block. Otherwise we'll be sending this style to everyone for no reason.